features/add_cart.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Add dish to cart
+  As a visitor
+  In order to select a dish that I´d like to order
+  I would like to place it in the cart
+
+  Background:
+   Given the following restaurants exist
+    | name        |
+    | NisseKebbab |
+   And the following menus exist
+    | name            | restaurant  |
+    | Nisses Takeaway | NisseKebbab |
+   And the following dishes exist
+    | name       | price | menu            |
+    | Kebabrulle | 180   | Nisses Takeaway |
+    | Falafel    | 190   | Nisses Takeaway |
+
+  Scenario: I buy a dish and put it in the cart
```

Comment:
I would argue that the scenario title does not fit what is going on in the scenario itself. I would like to see 2 scenarios (at least) for this feature. Adding one dish and adding 2 dishes.